### ISSUE TITLE
Deferred platform initialization tasks don't run until completion

### ISSUE THREAD

**Issue by nilox94** [BernardFW/bernard#92]
Deferred initialization tasks aren’t completed as the original event loop lifetime is too short.

In this example, we have an `asyncio.sleep` forcing the `_deferred_init` task to suspend early.
The first loop (in `start_all`) contains the tasks (and the non-deferred platform initialization coroutines) and finishes relatively fast.
On the other hand, the second loop executes the web server forever, but can’t execute foreign operations.

```python
import asyncio
from pathlib import Path
from unittest.mock import patch

from aiohttp import web

from bernard.cli._base import init_uvloop
from bernard.conf.utils import patch_conf
from bernard.engine.platform import SimplePlatform
from bernard.platforms import PlatformManager, start_all
from bernard.utils import run

transitions = []

module = Path(__file__).stem


class SlowInitPlatform(SimplePlatform):
    """
    This is a platform that will take some time to initialize.
    """

    async def _deferred_init(self):
        print("before suspend")
        await asyncio.sleep(0.01)
        print("after suspend")


PLATFORM_CONFIG = {
    "PLATFORMS": [
        {
            "class": f"{module}.SlowInitPlatform",
            "settings": {
                "name": "slow-init-platform",
            },
        },
    ],
    "DEFAULT_STATE": "bernard.engine.BaseState",
    "TRANSITIONS_MODULE": module,
}


if __name__ == "__main__":
    from bernard.server import app

    with patch_conf(PLATFORM_CONFIG):
        # recreate PlatformManager instance with the fresh config
        with patch("bernard.platforms.manager", PlatformManager()):
            # CLI entry point (like in bernard.cli._base.main)
            init_uvloop()
            run(start_all())                              # first loop
            web.run_app(app, host="127.0.0.1", port=8080) # second loop

```

Execution output:
```
before suspend
======== Running on http://127.0.0.1:8080 ========
(Press CTRL+C to quit)
```

**Comment by nilox94** [comment]
As I see it, we have two options:
1. Use a single loop in the application, and the tasks will finish during the idle period of the web server
2. Ensure first loop tasks run until completion before starting the web server

If there's no problem with the order of the operations, I'd suggest going with the first option since the second one is unnecessarily sequential.

**Comment by Xowap** [comment]
Hi there @nilox94, this will be fixed by #94
